src/main/java/com/robosh/ejournal/entity/admin/Admin.java

```diff
@@ -0,0 +1,51 @@
+package com.robosh.ejournal.entity.admin;
+
+import com.robosh.ejournal.entity.School;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.lang.Nullable;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
```

Comment:
add entity and columns names please

Reply:
I have added columns. Also, my task included creating the only entity, add column names it's yours)

Reply:
could you please do it ? because i`ve already done pull request for this task without admin